### ISSUE TITLE
Hostname issues

### ISSUE THREAD

**Issue by xor7589** [dopplertask/dopplertask#28]
When using in docker with hostname containing _ throws "java.lang.IllegalArgumentException: The character [_] is never valid in a domain name.".
Is this check necessary?

**Comment by feraswilson** [comment]
Yes, Domain names cannot contain an underscore character. Are you trying to do something specific?

**Comment by xor7589** [comment]
Yup, run this in host with underscore in its name on my lan in docker. No domains here.
Why assume that by default its run in domain?

**Comment by feraswilson** [comment]
Aha I see your point. I will have a look.
